Modules/Devices/src/Devices.Domain/Entities/Identities/IdentityDeletionProcess.cs

```diff
@@ -53,6 +53,13 @@ private void Approve(DeviceId createdByDevice)
         GracePeriodEndsAt = SystemTime.UtcNow.AddDays(IdentityDeletionConfiguration.LengthOfGracePeriod);
     }
 
+    private void Reject(DeviceId createdByDevice)
```

Comment:
The only reason I placed it above the public method it uses it and above the properties was because that was already the case with `Approve`. Should I place the private `Approve` below the public as well?

Reply:
Yes please. :)

Reply:
Done :)